modules/modify-rhsso-requests-limits.adoc

```diff
@@ -0,0 +1,30 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="modifying-rhsso-resource-requests-limits_{context}"]
+= Modifying RHSSO resource requests/limits
+
+[role="_abstract"]
+By default, the RHSSO container is created with default resource requests and limits as shown below:
+
+|===
+|*Resource* |*Requests* |*Limits*
+
+|CPU|500|1000m
+|Memory|512 Mi|1024 Mi
+
+|===
+
+You can modify the default resource requirements patching the Argo CD CR:
+
+[source,terminal]
+----
+$ oc -n openshift-gitops patch argocd openshift-gitops --type='json' -p='[{"op": "add", "path": "/spec/sso", "value": {"provider": "keycloak", "resources": {"requests": {"cpu": "512m", "memory": "512Mi"}, "limits": {"cpu": "1024m", "memory": "1024Mi"}} }}]'
+----
+
+[NOTE]
+====
+RHSSO created by the {gitops-title} only persists the changes that are made by the operator. Incase of RHSSO restarts, any additional configuration created by the Admin in RHSSO will be deleted.
```

Comment:
second sentence rephrase:

`If the RHSSO restarts, any additional configuration created by the Admin in RHSSO is deleted.`